kafka_redis_fs/fs-kafka-redis.ipynb

```diff
@@ -1,21 +1,42 @@
 {
```

Comment:
<p>why this was changed to load_project ? please use get_or_create_project as before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Changed</p>